satpy/etc/readers/hdfeos_l1b.yaml

```diff
@@ -21,7 +21,9 @@ datasets:
       250: {file_type: hdf_eos_data_250m}
       500: {file_type: hdf_eos_data_500m}
       1000: {file_type: hdf_eos_data_1000m}
-    calibration: reflectance
+    calibration:
+      reflectance:
+        units: "%"
```

Comment:
Do we know that the units coming out of this are `%` (0-~120) and not `1` (0 - ~1.2)?